### DISCUSSION TITLE
CDL model and UMi, UMa, and RMa applications

### DISCUSSION THREAD

**Discussion by EisakuSato-kke** [NVlabs/sionna#529] [Q&A] [answered]
I want to simulate mobile communications using MIMO and beamforming, so I am comparing sionna's CDL and UMi and other channel models. Especially for beam control and

I found it described that CDL model is for link level simulation and UMi, UMa and RMa are channel models for system level simulation.

I also understood that the CDL model does not require node coordinates, so the received power is normalized and the angle of arrival is unchangeable.

On the other hand, UMi, UMa, and RMa seem to calculate the appropriate path loss and direction of arrival from the node coordinates.

Based on the information so far, I was interested in what CDL model would be needed for the evaluation.

This is essentially a question about TR38.901 and perhaps should not be asked here.
However, I could not find any other forum that accepts such questions, so let me ask it here.

**Comment by jhoydis** [comment, marked as answer]
Hi @EisakuSato-kke,

You can think about TDL/CDL as single snapshots of realizations from the UMi/UMa/RMa models.
They are mainly used for testing and benchmarking of link-level algorithms but do not provide the richness of the UMI/UMa/RMa models.

I think that you should use the Uni/UMa/RMa for you work on beam control because the AoAs and AoDs are fixed in CDL, leading to probably uninteresting results.

A probably even better alternative could be the use of ray-traced channels via Sionna RT. The advantage would be that there is a clear relationship between the position of a user in a scene and the observed AoAs/AoDs (in other words you have perfect spatial consistency). You could also visualize beam directions to see if your results make sense. 

Hope that helps.
